src/Controller/EnMarche/SummaryManagerController.php

```diff
@@ -39,17 +38,13 @@ public function indexAction()
         $this->disableInProduction();
 
         $member = $this->getUser();
-        $summary = $this->get(SummaryManager::class)->getForAdherent($this->getUser());
-        $pathImage = 'images/'.$summary->getMemberUuid().'.jpg';
-        $signature = SignatureFactory::create($this->getParameter('kernel.secret'))->generateSignature($pathImage, []);
+        $manager = $this->get(SummaryManager::class);
+        $summary = $manager->getForAdherent($this->getUser());
+        $manager->setUrlProfilePicture($summary);
```

Comment:
IMHO this can be fully handled by the manger, since you're always getting the summary.
What about `$manger->setUrlProfilePicture(Adherent $user)`  ?
same way for deletion `$manger->remove(Adherent $user)`